assets/src/components/map.tsx

```diff
@@ -534,6 +539,11 @@ const Garages = ({ zoomLevel }: { zoomLevel: number }) => (
 )
 
 const Map = (props: Props): ReactElement<HTMLDivElement> => {
+  const [
+    {
+      searchPageState: { selectedVehicleId },
+    },
+  ] = useContext(StateDispatchContext)
```

Comment:
suggestion: This may be an issue on the mini-map, where the selected vehicle is stored as `state.selectedVehicleOrGhost?.id`

Consider passing the selectedVehicleId as a prop to the map so that it can be set on a case-by-case basis (`selectedVehicleId` for the mini map and old search map, `searchPageState.selectedVehicleId for the new map page)